tests/utils/futures.nim

```diff
@@ -0,0 +1,5 @@
+import chronos
+
+const
+  DURATION_TIMEOUT* = 1.seconds
+  DURATION_TIMEOUT_EXTENDED* = 1500.milliseconds
```

Comment:
This shouldn't be defined in an utils file. The constant names don't have any meaningful description and we don't have any requirement or consensus about the use of those values in the test suite. If you want to avoid magic numbers and repetition in your tests, it's better to define them for a particular test file.

Reply:
I would say `DURATION_TIMEOUT` is not a horrible name. It specifies the type (duration) and the context (timeout). Given that's the default measurement I'm using, I decided to leave the name as is. I believe the `EXTENDED` suffix can be easily understood from that.

The reason I'm specifying them in a file is precisely because I'm using those numbers widely across different files, so it felt a bit repetitive having to repeat them all over the test codebase.